src/tutorials/selfhosting/selfhost-ubuntu-sources-en.md

```diff
@@ -0,0 +1,524 @@
+
+
+    cozy     1h     IN         A     <your_server_IP>
+    *.cozy   1h     IN     CNAME     cozy
+
+Then install Nginx
+
+    sudo apt install -y nginx certbot
+
+Generate SSL certificate with certbot
+
+    DOMAIN=domain.example
+    EMAIL="<votre email>"
+    sudo certbot certonly --email "${EMAIL}" --non-interactive --agree-tos --webroot -w /var/www/html -d cozy.${DOMAIN} $(printf -- " -d %s.cozy.${DOMAIN}" home banks contacts drive notes passwords photos settings store)
+
+Create nginx reload script for your certificate to be reloaded each time it is automatically refreshed, every 3 monthes
+
+    cat <<EOF | sudo tee /etc/letsencrypt/renewal-hooks/deploy/reload-nginx.sh
+    #!/bin/bash
+    nginx -t -q && nginx -s reload
+    EOF
+    chmod 0755 /etc/letsencrypt/renewal-hooks/deploy/reload-nginx.sh
+
+Configure nginx
+
+    DOMAIN=domain.example
+    cat <<EOF | sudo tee /etc/nginx/sites-available/cozy.${DOMAIN} > /dev/null
+    server {
+        listen 80;
+        listen [::]:80;
+
+        root /var/www/html;
+        server_name *.cozy.${DOMAIN} cozy.${DOMAIN};
+
+        location /.well-known {
+            alias /var/www/html/.well-known;
+        }
+
+        location / {
+            return         301 https://\$host\$request_uri;
+        }
+    }
+
+    server {
+        listen 443 ssl http2;
+        listen [::]:443 ssl http2;
+        ssl_certificate /etc/letsencrypt/live/cozy.${DOMAIN}/fullchain.pem;
+        ssl_certificate_key /etc/letsencrypt/live/cozy.${DOMAIN}/privkey.pem;
+
+        server_name *.cozy.${DOMAIN} cozy.${DOMAIN};
+        access_log /var/log/nginx/cozy.${DOMAIN}.access.log;
+        error_log /var/log/nginx/cozy.${DOMAIN}.error.log;
+
+        add_header Strict-Transport-Security "max-age=31536000; includeSubDomains;";
+        client_max_body_size 1g;
+
+        location / {
+            proxy_pass http://localhost:8080;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade \$http_upgrade;
+            proxy_set_header Connection "upgrade";
+            proxy_set_header Host \$host;
+            proxy_set_header X-Forwarded-For \$remote_addr;
+        }
+    }
+    EOF
+    sudo ln -s ../sites-available/cozy.${DOMAIN} /etc/nginx/sites-enabled/
+    sudo systemctl reload nginx
+
+You can then test from your browser by visiting `https://cozy.domain.example` and you should see a page telling you this cozy instance doesn't exist yet. This is the sign that everything went well and the only part left is to create the instance.
+
+# Cozy instance creation
+
+Create your cozy instance
+
+    DOMAIN=domain.example
+    EMAIL=<votre email>
+    [[ -z "${COZY_PASS}" ]] && read -p "Cozy stack admin password: " -r -s COZY_PASS
+    COZY_ADMIN_PASSWORD="${COZY_PASS}" cozy-stack instances add --apps home,banks,contacts,drive,notes,passwords,photos,settings,store --email "${EMAIL}" --locale fr --tz "Europe/Paris" cozy.${DOMAIN}
```

Comment:
```suggestion
    COZY_ADMIN_PASSWORD="${COZY_PASS}" cozy-stack instances add --apps home,banks,contacts,drive,notes,passwords,photos,settings,store --email "${EMAIL}" --locale fr --en "Europe/Paris" cozy.${DOMAIN}
```

Reply:
I think you mean `--locale en` instead of `--en`

Beacause our main audience is currently in france, I prefer to leave the french locale and Paris timezone in the command example.

Anyway, the text below explain how to change that.

Reply:
Oh sh** yes this is what I meant 😵‍💫 . I saw just after the next line but kept the comment to see what do you though about it